purchases/build.gradle

```diff
@@ -36,6 +36,9 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation project(":common")
+    implementation project(":utils")
+    api project(":public")
```

Comment:
the main difference between `implementation` and `api` is that anyone depending on this module `purchases` (meaning anyone using our SDK) will have access to the classes in the `public` module